### DISCUSSION TITLE
Elasticsearch 8

### DISCUSSION THREAD

**Discussion by Prebiusta** [Netflix/conductor-community#135] [Q&A]
Hello, 

I stumbled upon the problem when trying to index data into Elasticsearch 8.4. Are you planning on implementing indexing for the newest Elasticsearch? It's quite unfortunate that I cannot use the ES cluster we are running (version 8.4) and I am forced to create another instance of ES6 just for the conductor.

Please let me know what are the options here

**Comment by jxu-nflx** [comment]
Hello @Prebiusta we don't have plan to support ES 8 yet. Feel free to contribute to the[ conductor-community](https://github.com/Netflix/conductor-community/tree/main/index ) repo
